pudl_scrapers/spiders/eia860m.py

```diff
@@ -0,0 +1,123 @@
+"""Spider for 860 M."""
+# -*- coding: utf-8 -*-
+from pathlib import Path
+import scrapy
+from scrapy.http import Request
+
+import pudl_scrapers
+
+
+class Eia860MSpider(scrapy.Spider):
+    """Spider for Monthly EIA 860."""
+
+    name = 'eia860m'
+    allowed_domains = ['www.eia.gov']
+
+    def __init__(self, year=None, month=None, *args, **kwargs):
+        """Spider for scrapping EIA 860 Monthly."""
```

Comment:
Can you document the arguments here? Or in the class-level docstring? What types are year and month? What are acceptable values? What if someone gives a bad value for month?